docs/developer/architecture/snmp.md

```diff
@@ -133,193 +132,193 @@ helm install datadog-monitoring --set datadog.apiKey=<YOUR_API_KEY> -f cluster-a
 
 ??? example "Example cluster-agent-values.yaml"
 
-    ```yaml
```

Comment:
This is also already correct, we don't want to cluster the page so we put it in a https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details